config/public.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
We can remove this as we will have to overwrite this anyway

Reply:
Done!